activation-config.yaml

```diff
@@ -35,6 +35,8 @@ googleCloud:
     # Cron schedule for Cloud Scheduler instance
     schedulerSchedule: "* * * * *" 
   logs:
+    # Name of Google Secret Manager Secret containing Dynatrace logs ingest URL
```

Comment:
I'm a little confused now - `activation-config.yaml` is used by the script for function deployment (not kubernetes setup) - as we added support for logs only for kubernetes setup why we have logs section in this file? What am I missing?

Reply:
You are right, I did too much changes :)

Reply:
It seem entire `logs:` section is not needed - I'm removing it